tests/migrations/test_operations.py

```diff
@@ -1489,6 +1492,24 @@ def test_remove_index(self):
         self.unapply_operations("test_rmin", project_state, operations=operations)
         self.assertIndexExists("test_rmin_pony", ["pink", "weight"])
 
+    def test_add_constraint(self):
+        """Test the AddConstraint operation."""
+        project_state = self.set_up_test_model('test_addconstraint')
+
+        where = WhereNode([GreaterThan(Col('pink', models.F('pink')), 2)])  # Q(pink__gt=2)
```

Comment:
Long term, I want to be able to use `Q(pink__gt=2)` here, but for now using this avoids the need to implement `Q.as_sql`.